apps/communications/contacts/locales/contacts.en-US.properties

```diff
@@ -59,13 +65,14 @@ anniversary               = Anniversary
 
 # Contact action buttons
 addPhoto       = Add Picture
-addPhone       = Add Phone
```

Comment:
as mentioned number of times earlier - it's **deprecated to change ANY existing l10n strings**. If you need new text you shall create new l10n id, otherwise such changes wouldn't be detected by l10n team and translations wouldn't be updated at all.